### PR TITLE
added the contributors copyright license with description.

### PR DESCRIPTION
**Pull request by mrshirts**

### CODE REVIEW

File: _author_instructions/01_policies.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ Additionally, articles should include clear links to, and mention of, the releva
 
 For an example of an article hosted on GitHub in this style (though not a LiveCoMS article), see [the following perpetual review](https://github.com/MobleyLab/benchmarksets).
 
+### Copyright license
+
+In order to publish the article, LiveCoMS requires that authors sign (an additional contributor license)[LiveCoMS_contributor_license.pdf] to clarify LiveCoMS rights to publish the article, regardless of the license the author uses in the their own release of the article on GitHub.  This document also to confirms that the authors grant the publisher the right to distribute their article and that the authors are legally entitled to do so for all the material in the article.
```
Comment (dmzuckerman) [inline _author_instructions/01_policies.md]: typo: "the their".  Otherwise good

File: _author_instructions/01_policies.md [hunk 2]
```diff
@@ -68,6 +68,12 @@ Additionally, articles should include clear links to, and mention of, the releva
 
 For an example of an article hosted on GitHub in this style (though not a LiveCoMS article), see [the following perpetual review](https://github.com/MobleyLab/benchmarksets).
 
+### Copyright license
+
+In order to publish the article, LiveCoMS requires that authors sign (an additional contributor license)[LiveCoMS_contributor_license.pdf] to clarify LiveCoMS rights to publish the article, regardless of the license the author uses in the their own release of the article on GitHub.  This document also to confirms that the authors grant the publisher the right to distribute their article and that the authors are legally entitled to do so for all the material in the article.  
+
+This agreement is a license, *not a copyright transfer*; authors retain the copyright for their works, and therefore authors are not restricted in their own use or distribution of their work in any way by this document.
```
Comment (dmzuckerman) [inline _author_instructions/01_policies.md]: Good.

**Comment by mrshirts** [comment]
> One small note: Is it your intention to have the one deposited say "DRAFT"? Because it does. It's perhaps odd since I would think we'd want the final one up...?

Hah, I missed that.   I will talk with Maggie and get a clean final draft to post before the PR gets merged.

File: _author_instructions/01_policies.md (hunk 1 above)
Comment (mrshirts) [reply _author_instructions/01_policies.md]: Fixed!

**Comment by mrshirts** [comment]
OK, updated the license, merging now.

**Pull request merged**